extensions/amp-story/0.1/amp-story.js

```diff
@@ -355,6 +356,8 @@ export class AmpStory extends AMP.BaseElement {
   buildCallback() {
     this.assertAmpStoryExperiment_();
 
+    installMediaPoolServiceForDoc(this.getAmpDoc());
```

Comment:
Should we still be explicitly instantiating a `MediaPool` in the constructor above?  We also now instantiate `MediaPool` in [amp-story-page](https://github.com/ampproject/amphtml/blob/ecc62dd1c4c22e0bb309e2965a7ad94ed23dd9cc/extensions/amp-story/0.1/amp-story-page.js#L162)

Reply:
The service provides the pools being instantiated. Since the pool is always bound by a node itself it can find the right instance.